openapi-v2.yaml

```diff
@@ -2445,6 +2493,50 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
+  /groups/{group_namespace}/{group_name}/content_activity:
+    parameters:
+      - name: group_namespace
+        type: string
+        in: path
+        required: true
+        description: The namespace of the group
+      - name: group_name
+        type: string
+        in: path
+        required: true
+        description: The unique name or id of the group
+      - name: page
+        in: query
+        description: pagination offset
+        type: integer
+        required: false
+      - name: per_page
+        in: query
+        description: pagination limit
+        type: integer
+        required: false
```

Comment:
Do we need any controls for the amount of data returned inside each asset’s activities, to go with the pagination here of how many assets we return in a batch?